--- __init__.py 2015-05-01 19:38:01.000000000 +0530
+++ __init__.py   2015-05-01 19:37:44.144733110 +0530
@@ -165,10 +165,11 @@
         run.
         """
         duration = self._duration(self._run_start)
+
         self._stream.write('<testsuite errors="%d" failures="%d" name="" '
-            'tests="%d" time="%0.3f">\n' % (len(self.errors),
+            'skipped="%d" tests="%d" time="%0.3f">\n' % (len(self.errors),
             len(self.failures) + len(getattr(self, "unexpectedSuccesses", ())),
-            self.testsRun, duration))
+            len(self.skipped),self.testsRun, duration))
         self._stream.write(''.join(self._results))
         self._stream.write('</testsuite>\n')
 

